cmd-record.c

```diff
@@ -108,8 +174,8 @@ static void setup_child_environ(struct opts *opts, int pfd)
 	old_libpath = getenv("LD_LIBRARY_PATH");
 	if (old_libpath) {
 		size_t len = strlen(buf) + strlen(old_libpath) + 2;
-		char *libpath = xmalloc(len);
 
+		libpath = xmalloc(len);
 		snprintf(libpath, len, "%s:%s", buf, old_libpath);
 		setenv("LD_LIBRARY_PATH", libpath, 1);
 		free(libpath);
```

Comment:
I thought about this logic again, and now I think we don't need to set `LD_LIBRARY_PATH` anymore since it uses a full path for `LD_PRELOAD`.

Reply:
I agree that. I also agonized it because it seems to be duplicated.
I'll fix it based on your comment.